packages/react-theme-provider/src/common/isConformant.ts

```diff
@@ -0,0 +1,10 @@
+import { isConformant as baseIsConformant, IsConformantOptions } from '@fluentui/react-conformance';
+
+export function isConformant(testInfo: Omit<IsConformantOptions, 'componentPath'>) {
+  const defaultOptions = {
```

Comment:
```suggestion
Same here
```suggestion
export function isConformant<TProps = {}>(
  testInfo: Omit<IsConformantOptions<TProps>, 'componentPath'> & { componentPath?: string },
) {
  const defaultOptions: Partial<IsConformantOptions<TProps>> = {
```
```